articles/truebit_incentive/truebit_light_incentives.tex

```diff
@@ -120,35 +120,35 @@
 correct computations/transactions to be included.
 
 This article wants to specifically address the Dogecoin-Ethereum bridge, which requires blocks from the Dogecoin
-blockchain to be verified inside an Ethereum smart contract. This verification is too expensive to be done directly,
-and because of that, we utilize TrueBit to take the computation off-chain.
+blockchain to be verified inside an Ethereum smart contract. This verification is expensive when done directly,
+and because of that, we are incentivized to utilize TrueBit to take the computation off-chain.
 
 Having said that, all analyses are equally applicable to bridges between blockchains where the availability of blocks
 can be reasonably assumed. This means that it can be used to e.g.\ offload processing volume from the main Ethereum
 chain to another blockchain (which might even be a proof-of-authority chain) as long as all participants in that chain
 rightfully assume that block data is available to all potential challengers.
 
-We call TrueBit-light the protocol that does not make an effort to bring honest participants to the network.
+We call the version of the protocol that does not make an effort to bring honest participants to the network TrueBit-light.
 We assume that an honest participant is present who is altruistic to a certain degree. This includes keeping up
 with the Ethereum network, paying the gas fee and having a certain amount of money to pay for an initial deposit.
 
 We also simplify the system to a degree to only allow one parallel task. This might be extended to a constant amount
 of tasks but would also make the presentation here more complex.
 
-All timeouts are lower bounds and have to be extended in case of Ethereum network congestion. This means that if you want
-to claim a timeout to take effect, you have to provide a proof that several previous blocks had enough capacity to
+All timeouts have lower bounds and have to be extended in the case of network congestion. This means that if you want
```

Comment:
I think this whole paragraph has to be extended, but I also did not yet include this in the formalization. And yeah, as it reads, I think you would probably assume that you have to manually extend the timeouts, but they are actually extended automatically. Whether they are or have lower bounds is probably debatable (depending on whether you see the timeout as the timespan as a value or the event that happens when the time runs out).